styles/colors-silicon-night.less

```diff
@@ -11,8 +11,8 @@
 @agate-bg-color: black;
 @agate-bg-image: none;
 @agate-bg-image2: none;
-@agate-text-color: @agate-foreground;
-@agate-title-text-color: @agate-foreground;
+@agate-text-color: @agate-background;
```

Comment:
please change to a @agate-white; @agate-background will be overriden by FORTECH-26 with the black value. this means the text color there will be a black text on a black background